### DISCUSSION TITLE
Is it possible to embed a toodle?

### DISCUSSION THREAD

**Discussion by xrd** [ianthehenry/toodle.studio#11] [Q&A] [answered]
Is there a way to export a toodle and host it inside another web page? Can I pull out the relevant wasm and associated JavaScript files and then configure an app at runtime with my Janet code and run just the canvas output frame?

**Comment by ianthehenry** [comment, marked as answer]
Yeah! Go ahead! I suspect that copying the source and hacking it down to the minimal subset is gonna be your best bet -- there's no way to like export an embeddable version out of the box.
